robolectric/src/test/java/org/robolectric/shadows/ShadowAccessibilityNodeInfoTest.java

```diff
@@ -3,18 +3,22 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.Assert.assertEquals;
 
+import android.R.anim;
 import android.graphics.Rect;
+import android.os.Bundle;
 import android.os.Parcel;
 import android.view.accessibility.AccessibilityNodeInfo;
+import android.view.accessibility.AccessibilityNodeInfo.AccessibilityAction;
 
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.robolectric.TestRunners;
+import org.robolectric.annotation.Config;
 import org.robolectric.internal.ShadowExtractor;
 
-@RunWith(TestRunners.WithDefaults.class)
+@RunWith(TestRunners.MultiApiWithDefaults.class)
 public class ShadowAccessibilityNodeInfoTest {
 
   private AccessibilityNodeInfo node;
```

Comment:
ShouldHaveClonedCorrectly (nit: capitalization) is a pretty weak test. Since it's using defaults, it would probably pass even if you deleted big chunks of clone.